src/Process/Pool/Logger/Message/Process/FromProcessInterfaceInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\Process;
+
+use Neighborhoods\Kojo\ProcessInterface;
+
+interface FromProcessInterfaceInterface extends \JsonSerializable
+{
+
+    public function jsonSerialize();
```

Comment:
Nitpick but you don't need to define this because we're extending `\JsonSerializable`